Shoko.Server/API/SignalR/Models/EpisodeInfoUpdatedEventSignalRModel.cs

```diff
@@ -0,0 +1,32 @@
+using Shoko.Plugin.Abstractions;
+using Shoko.Server.Models;
+using Shoko.Server.Repositories;
+
+namespace Shoko.Server.API.SignalR.Models
+{
+    public class EpisodeInfoUpdatedEventSignalRModel
+    {
+        public EpisodeInfoUpdatedEventSignalRModel(EpisodeInfoUpdatedEventArgs eventArgs)
+        {
+            var series = RepoFactory.AnimeSeries.GetByAnimeID(eventArgs.AnimeInfo.AnimeID);
+            EpisodeID = ((SVR_AnimeEpisode)eventArgs.EpisodeInfo).AnimeEpisodeID;
+            SeriesID = series.AnimeSeriesID;
+            GroupID = series.AnimeGroupID;
+        }
```

Comment:
This as a whole seems like a N+1 issue, why do we need to have this lookup - is it better to have it in the EventArgs model itself?

Reply:
The plugin abstraction uses the SVR_AniDB_Anime for it's IAnime implementation, The SVR_AniDB_Anime model does not contain the shoko ids. In my opinion if we want to fix having to look up the series from the anime id then we need to better align the plugin abstractions with how shoko internally models the data.

Reply:
That would ultimately solve the N+1 issue that has been created here, because what is now happening is, we look up the details, create the interface details, pass it down to the event and look it back up.

Ultimately this becomes a potential problem for performance issues, and we should be grabbing it only once.

If you need more detail on the concept of N+1 issues: https://medium.com/doctolib/understanding-and-fixing-n-1-query-30623109fe89 while in rails, the concepts are entirely the same, and will become more evident when we move to EF, and don't really have the caches in the long run.